python/lsst/ip/isr/isrStatistics.py

```diff
@@ -115,6 +117,43 @@ class IsrStatisticsTaskConfig(pexConfig.Config):
         default=[0, 5, 16, 50, 84, 95, 100],
     )
 
+    doBiasShiftStatistics = pexConfig.Field(
+        dtype=bool,
+        doc="Measure number of image shifts in overscan?",
+        default=False,
+    )
+    biasShiftFilterOrder = pexConfig.Field(
+        dtype=int,
+        doc="Filter order for highpass filter.",
+        default=5,
+    )
+    biasShiftCutoff = pexConfig.Field(
+        dtype=float,
+        doc="Cutoff frequency for highpass filter.",
+        default=1.0/15.0,
+    )
+    biasShiftWindow = pexConfig.Field(
+        dtype=int,
+        doc="Filter window size for highpass filter.",
+        default=30,
+    )
+    biasShiftThreshold = pexConfig.Field(
+        dtype=float,
+        doc="Threshold for bias shift detection.",
```

Comment:
What units is this?

Reply:
It's a signal/noise threshold.  I've updated the doc to indicate that.